sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/impl/action/ActionHandler.scala

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.scalasdk.impl.action
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import com.akkaserverless.scalasdk.action.Action
+import com.akkaserverless.scalasdk.action.ActionContext
+import com.akkaserverless.scalasdk.action.MessageEnvelope
+import com.akkaserverless.scalasdk.impl.action.ActionHandler.HandlerNotFound
+
+object ActionHandler {
+  case class HandlerNotFound(commandName: String) extends RuntimeException
```

Comment:
For views I skipped this one, and use the javasdk HandlerNotFound. The reasons were:
1. then I didn't have to catch convert and throw
2. it's not exposed to users anyway (only in generated Handler code)